### DISCUSSION TITLE
Automatic night charging

### DISCUSSION THREAD

**Discussion by DavidKarlas** [tillsteinbach/VWsFriend#116] [Show and tell]
Hi,

first of all thank you for all the effort, I'm using VWsFriend to store and visualize data.
But recently I also wanted to automatically charge car during night because we have 2 tariffs prices in Slovenia(22:00-6:00 and weekends is cheaper).
So what I did is this:
```
crontab -e
```
and added this two lines:
```
0      22      *       *       *       weconnect-cli set /vehicles/WVWZZXXXXXXXXXXXX/controls/charging start
0      6       *       *       MON-FRI weconnect-cli set /vehicles/WVWZZXXXXXXXXXXXX/controls/charging stop
```

Only downside is, that when I plug my car in when I want to charge during week I need to stop charging manually via mobile app.

**Comment by tillsteinbach** [comment]
Interesting use-case! I hope it is reliable enough so you don’t stand with an empty car one morning. Some times the VW servers have problems.

**Reply by DavidKarlas** [reply]
Slovenia is small country, even with 50% charge I can travel half country :D
Highway DC chargers in worst case scenario...
I usually only use 5% of charge per day sometimes 15%.. I usually only plug it when its under 50% and it goes up to 80%...
So I'm not too worried...
So far Schuko that came with car is enough... Tnx to this tool :)

**Comment by Julien4135** [comment]
It would not be possible if you use the HomeKit app on iOS to create an automation can be ?

**Reply by DavidKarlas** [reply]
I do use iPhone, but didn't bother setting up HomeKit since I wasn't sure if I could do this, thank you for idea.
I ordered https://shop.go-e.co/go-eCharger-HOME-11-kW-A_1 last week since sometimes on weekends when I do long trip in morning and want to do another at night is a bit hard with 2kW charging, and that charger should take care of all this automation...

**Reply by Julien4135** [reply]
Ah yes it's true that 2kW is a long time to charge! it is going better in 11kW now :)
For HomeKit it is to be checked because it is not necessarily easy to find the right program to do on HomeKit.
